app/views/landing_page/blocks/_statistics.html.erb

```diff
@@ -18,4 +18,7 @@
   height: height,
   table_direction: "vertical",
   point_size: block.data["point_size"],
+  h_axis_format: block.data["h_axis_format"],
+  v_axis_format: block.data["v_axis_format"],
```

Comment:
I wonder if we should make these `x_axis_format` and `y_axis_format` to match `x_axis_label` and `y_axis_label`?